rdt-open-reader/src/main/java/io/ona/rdt/presenter/PatientRegisterFragmentPresenter.java

```diff
@@ -69,7 +72,7 @@ public String mainSelect(String tableName, String mainCondition) {
     }
 
     private String[] mainColumns(String tableName) {
-        String[] columns = new String[]{tableName + "." + "relationalid", tableName + "." + FIRST_NAME, tableName + "." + LAST_NAME, tableName + "." + AGE, tableName + "." + SEX, tableName + "." +  PATIENT_ID};
+        String[] columns = new String[]{tableName + DOT + "relationalid", tableName + DOT + FIRST_NAME, tableName + DOT + LAST_NAME, tableName + DOT + AGE, tableName + DOT + SEX, tableName + DOT +  PATIENT_ID, tableName + DOT + Constants.DBConstants.DOB};
```

Comment:
I know this is cleanup for existing code, but can we use a string join here?

Reply:
fixed.